.github/workflows/push.yml

```diff
@@ -0,0 +1,230 @@
+name: Pipeline
```

Comment:
Will the same name work?

Reply:
Yes, we are not running both workflows at the same time
1. build workflow will run for PR
2. pus workflow will run after the PR merge